emscripten.py

```diff
@@ -599,7 +599,7 @@ def make_emulated_param(i):
     if metadata['simdUint32x4']:
       simdinttypes += ['Uint32x4']
       simdintfloatfuncs += ['fromUint32x4Bits']
-    if metadata['simdInt32x4']:
+    if metadata['simdInt32x4'] or settings['SIMD']: # Always import Int32x4 when building with -s SIMD=1, since memcpy is SIMD optimized.
```

Comment:
Yes, it can. It is possible that the application itself does not use Int32x4 SIMD type at all so SIMD=1 and simdInt32x4=0. Also it is possible to be the other way around, simdInt32x4=1 and SIMD=0, because simdInt32x4 is set to one when compiled .bc code is detected to contain SIMD constructs (the .bc could have been compiled earlier with different flags), but that is orthogonal to what the user set on the command line for -s SIMD=1.